### ISSUE TITLE
EDA design rules

### ISSUE THREAD

**Issue by mlp6** [mlp6/Fundamentals-BME-Design#30]
From @sphaerobolus "I’m thinking it would be good to give students DRCs for AD and Eagle. I’m thinking 20 mils as the smallest unit for almost every feature. (This will cause some SMT components to error, so probably should be tweaked.) I know we can share a DRU file for Eagle. I’m not yet sure how to do it in AD outside of sharing a .pcbdoc.  Is sharing design rules something you were already headed toward?"

**Comment by mlp6** [comment]
@sphaerobolus yes, definitely given our experience this semester!  Maybe have different sets of rules depending on the fabrication process?  I think / thought that the design rules can be tailored by certain component types too, which could work around the SMT issue with larger widths.

**Comment by sphaerobolus** [comment]
In Eagle, in an open board, go to Tools->DRC. Under that, the tabs

Clearance
Distance
Sizes
Annular Ring
Shapes
Supply

all are more or less relevant. Most of the changes I'm looking at are under Clearance and Sizes. First rough DRC is in BME_in_house.dru under Resources.
